README.md

```diff
@@ -193,13 +193,24 @@ A full list of tuning and config parameters is below. See the field description
             "num": 20,
             "seed": 3,
             "recsModel": "all",
-			"backfillField": {
-				"name": "popRank"
-  				"backfillType": "popular",
-  				"eventNames": ["buy", "view"],
-  				"duration": "3 days",
-  				"endDate": "ISO8601-date" //most recent date to end the duration
-  			},
+            "backfills": [
+                {
+                    "name": "popRank"
+                    "type": "popular", // or "trending" or "hot"
+                    "eventNames": ["buy", "view"],
+                    "duration": "3 days",
+                    "endDate": "ISO8601-date" // most recent date to end the duration
+                },
+                {
+                    "name": "uniqueRank"
+                    "type": "random",
+                    "eventNames": ["buy", "view"]
+                },
```

Comment:
Why are eventNames needed here? random must work on any item existing in the EventServer event those with no events. In fact this is because there is no way to get new items into recommendations since they do not have events and so never appear in "popular".

The user will $set the item to create an object in the EventServer, then is will get a random ranking even with no events.